README.md

```diff
@@ -21,7 +21,15 @@ and bound-constrained optimization
 This package provides an implementation of four classic algorithms for unconstrained/bound-constrained nonlinear optimization:
 
 - `lbfgs`: an implementation of a limited-memory BFGS line-search method for unconstrained minimization;
+    > Boggs, P. T., & Byrd, R. H. (2019). Adaptive, Limited-Memory BFGS
+    > Algorithms for Unconstrained Optimization. SIAM Journal on
+    > Optimization, 29(2), 1282-1299.
+    > DOI: [10.1137/16M1065100](https://doi.org/10.1137/16M1065100)
+    
 - `R2`: a first-order quadratic regularization method for unconstrained optimization;
+  > Monnet, Dominique & Orban, Dominique. (2023). A Multi-Precision Quadratic
+  > Regularization Method for Unconstrained Optimization with Rounding Error Analysis.
+  > DOI: [10.13140/RG.2.2.22068.01926.](http://dx.doi.org/10.13140/RG.2.2.22068.01926) 
```

Comment:
The version of R2 here is not the one described in that paper. One reference for R2 is https://link.springer.com/article/10.1007/s10107-016-1065-8 (R2 is a special case).

Reply:
I added the citation for the suggested reference.